interfaces/cython/cantera/composite.py

```diff
@@ -955,14 +957,23 @@ def read_csv(self, filename):
         using `restore_data`. This method allows for recreation of data
         previously exported by `write_csv`.
         """
-        # read data block and header separately
-        data = np.genfromtxt(filename, skip_header=1, delimiter=',')
-        labels = np.genfromtxt(filename, max_rows=1, delimiter=',', dtype=str)
-
-        data_dict = OrderedDict()
-        for i, label in enumerate(labels):
-            data_dict[label] = data[:, i]
-
+        if version.LooseVersion(np.__version__) < version.LooseVersion("1.14"):
```

Comment:
It looks like NumPy has the built-in capability to do this kind of comparison. I went searching because I was wondering if they had a tuple we could use instead of a string... anyhow, this should work:
```suggestion
        if np.lib.NumpyVersion(np.__version__) < "1.14.0":
```
and avoids the `distutils` import.

Reply:
Nice - I wasn’t aware of this!